tests/test_terrain.py

```
@@ -0,0 +1,122 @@
+        
+        # Act & Assert
+        # Points in landing zone
+        self.assertTrue(terrain.is_point_in_landing_zone(start))
+        self.assertTrue(terrain.is_point_in_landing_zone(middle))
+        self.assertTrue(terrain.is_point_in_landing_zone(end))
+        
+        # Points outside landing zone
+        self.assertFalse(terrain.is_point_in_landing_zone(start - 1))
+        self.assertFalse(terrain.is_point_in_landing_zone(end + 1))
+    
+    def test_small_screen_adaptation(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_small_screen_adaptation`

Add return type annotation: `None`

(ANN201)